src/new-client/src/App/Trades/TradesCore.tsx

```diff
@@ -7,28 +7,48 @@ import { TradesHeader } from "./TradesHeader"
 import { tableTrades$ } from "./TradesState"
 import { createSuspenseOnStale } from "@/utils/createSuspenseOnStale"
 import { isBlotterDataStale$ } from "@/services/trades"
+import { useCallback, useEffect, useRef, useState } from "react"
 
 const TradesStyle = styled.div`
   height: 100%;
   width: 100%;
   color: ${({ theme }) => theme.core.textColor};
   font-size: 0.8125rem;
+  position: relative;
 `
 
 const SuspenseOnStaleData = createSuspenseOnStale(isBlotterDataStale$)
+const Trades: React.FC = () => {
+  const [height, setHeight] = useState(0)
+  const [width, setWidth] = useState(0)
+  const ref = useCallback((node) => {
+    if (node) {
+      const resizeObserver = new ResizeObserver((entry) => {
+        const { height, width } = entry[0].contentRect
+        setHeight(height)
+        setWidth(width)
+      })
+      resizeObserver.observe(node)
+    }
```

Comment:
Need to add a disconnect/unobserve here

Reply:
How can I unobserve a node when this already dissapeared from the dom?

```
  const ref = useCallback((node) => {
    let resizeObserver = new ResizeObserver(() => {})
    if (node) {
      resizeObserver = new ResizeObserver((entry) => {
        const { height, width } = entry[0].contentRect
        setHeight(height)
        setWidth(width)
      })
      resizeObserver.observe(node)
    }else{
      //Can't unobserve the node since it's undefined
      // resizeObserver.unobserve(node)
      //Don't know if the disconnect would do the trick
      // resizeObserver.disconnect()
    }
  }, [])
```

Reply:
disconnect should work
https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver/disconnect